modules/sand/src/test/ts/browser/HtmlElementTest.ts

```diff
@@ -1,11 +1,53 @@
-import { assert, UnitTest } from '@ephox/bedrock-client';
-import { SandHTMLElement } from 'ephox/sand/api/Main';
-
-UnitTest.test('HtmlElementTest', () => {
-  const span = document.createElement('div');
-  assert.eq(false, SandHTMLElement.isPrototypeOf(null));
-  assert.eq(false, SandHTMLElement.isPrototypeOf(undefined));
-  assert.eq(false, SandHTMLElement.isPrototypeOf('a string'));
-  assert.eq(false, SandHTMLElement.isPrototypeOf({}));
-  assert.eq(true, SandHTMLElement.isPrototypeOf(span));
+import { context, describe, it } from '@ephox/bedrock-client';
+import { assert } from 'chai';
+
+import * as SandHTMLElement from 'ephox/sand/api/SandHTMLElement';
+
+describe('HtmlElementTest', () => {
+  context('isPrototypeOf', () => {
+    it('non elements should be false', () => {
```

Comment:
Shouldn't it have `TINY-7374` in the test label?

Reply:
No, this is the existing functionality and not related to this change. There's only the one test that's related to this change.